jenkins/jobs/lxd-test-snap.yaml

```diff
@@ -77,11 +77,8 @@
 
     builders:
     - shell: |-
-        if [ "${{dist}}" = "ubuntu-core-16" ] || [ "${{dist}}" = "ubuntu-core-18" ] || [ "${{dist}}" = "ubuntu-core-20" ]; then
-        cd ~/lxc-ci
-        else
-        cd /lxc-ci
-        fi
+        cd ~/lxc-ci 2>/dev/null || cd /lxc-ci
+        pwd
```

Comment:
why the pwd call?

Reply:
And I'd prefer `cd /lxc-ci || cd ~/lxc-ci` personally as a random `lxc-ci` in the home dir should not be select over the standard /lxc-ci

Reply:
`pwd` was to leave a visible trace of which dir was used, I dropped it.